gui/mozregui/wizard.py

```diff
@@ -105,6 +114,40 @@ class NightliesPage(WizardPage):
     FIELDS = {"start_date": "start_date", "end_date": "end_date"}
     ID = 1
 
+<<<<<<< HEAD
```

Comment:
Yep, I think this will do the job - commit rebase and push -f. Be sure you only make a patch that do what you intend here (and not some other stuff that comes from the other branch).